CHANGELOG.rst

```diff
@@ -17,6 +17,8 @@ Changed
 * Install pack with the latest tag version if it exists when branch is not specialized.
   (improvement) #4743
 * Implement "continue" engine command to orquesta workflow. (improvement) #4740
+* Update various internal dependencies to latest stable versions (apscheduler, eventlet,
+  kombu, amqp, pyyaml, mongoengine, python-gnupg, paramiko, tooz, webob, bcrypt). #4767
```

Comment:
Makes sense to include a small notes that mongo led to perf improvements, considering you've tested it.